pages/common/install.md

```diff
@@ -0,0 +1,24 @@
+# install
+
+> Copy files and set attributes.
+> Similar to `cp`, as well as control the attributes of destination files.
+
+- Copy files to destination:
+
+`install {{path/to/source}} {{path/to/destination}}`
+
+- Copy files to destination, setting their ownership:
+
+`install -o {{user}} {{path/to/source}} {{path/to/destination}}`
+
+- Copy files to destination, setting their group ownership:
+
+`install -g {{user}} {{path/to/source}} {{path/to/destination}}`
+
+- Copy files to destination, setting their permissions:
+
+`install -m {{mode}} {{path/to/source}} {{path/to/destination}}`
```

Comment:
Maybe a concrete example would make usage more evident. I'd favor an alphabetical mode example :) Note that if you do this, the word "mode" should be included in the description, since it wouldn't appear in the token itself.